scripts/webpack.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+webpack
+RESULT=$?
```

Comment:
maybe do "#!/bin/sh -e" instead? That would accomplish the same thing, and also complain if the ln command fails.